--- aircrack-ng-1.2-rc2/common.mak.orig	2015-11-17 13:23:14.717899992 +0100
+++ aircrack-ng-1.2-rc2/common.mak	2015-11-17 13:24:33.161433664 +0100
@@ -118,7 +118,7 @@ REVISION	= $(shell $(AC_ROOT)/evalrev $(
 REVFLAGS	?= -D_REVISION=$(REVISION)
 
 OPTFLAGS        = -D_FILE_OFFSET_BITS=64
-CFLAGS          ?= -g -W -Wall -O3
+CFLAGS          ?= -g -W -Wall $(PLDFLAGS)
 CFLAGS          += $(OPTFLAGS) $(REVFLAGS) $(COMMON_CFLAGS)
 
 prefix          = /usr/local
