--- aircrack-ng-1.2-rc2/scripts/airmon-ng.orig	2015-11-24 12:54:48.642540223 +0100
+++ aircrack-ng-1.2-rc2/scripts/airmon-ng	2015-11-24 12:57:14.755185878 +0100
@@ -236,7 +236,7 @@ startMac80211Iface() {
 	IW_ERROR=$(iw phy ${PHYDEV} interface add ${1}mon type monitor 2>&1 | grep "nl80211 not found")
 	if [ x${IW_ERROR} = "x" ]
 	then
-		sleep 1s
+		sleep 1
 		setChannelMac80211 ${1}mon
 		printf "\n\t\t(mac80211 monitor mode vif enabled for [${PHYDEV}]${1} on [${PHYDEV}]${1}mon)\n"
 	else
@@ -1095,7 +1095,7 @@ do
 
 		#why, dear god why is there a random newline here?
 		printf "\n"
-		sleep 1s
+		sleep 1
 		continue
 	fi
 done
